<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="playerLabel">
     <property name="text">
      <string>Player's Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="playerLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="charLabel">
     <property name="text">
      <string>Character's Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="charLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dexLabel">
     <property name="text">
      <string>Dexterity score</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dexLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
